### PR TITLE
added manual release on active claims and loading indications

### PR DESCRIPTION
**Pull request by rycastaneda**

### CODE REVIEW

File: app/scripts/config.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ module.exports = {
         components: './components/'
     },
     url: {
-        api: 'http://api.tunes.tm',
-        // api: 'http://dev.music.tm:3000',
+        // api: 'http://api.tunes.tm',
+        api: 'http://dev.music.tm:3000',
```
Comment (sibayanjasper) [inline app/scripts/config.js]: Do not change api config

File: app/scripts/components/claims/item.js [hunk 2]
```diff
@@ -68,14 +73,17 @@ var React = require('react'),
...
+                action_ref = 'release_btn' + this.props.claim_id;
                 action = (
                         <td className="" data-column-title=''>
-                                <button className="filter-btn btn waves-effect waves-light"  onClick={this.handleManualRelease}>Release</button>
+                                <button className="filter-btn btn waves-effect waves-light" ref={action_ref} onClick={this.handleManualRelease}>Release</button>
                         </td>
                     );
             }
 
+            console.log("this.props.status", this.props.status);
```
Comment (sibayanjasper) [inline app/scripts/components/claims/item.js]: Remove logs

**Comment by sibayanjasper** [comment]
Show release button only for active claims, hide otherwise

**Comment by rycastaneda** [comment]
@sibayanjasper  updated, added a loading bar

**Pull request merged**